tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestAssemblyInfoRunAssemblyInitializeIntegration.cs

```diff
@@ -83,4 +93,11 @@ internal static CallTargetReturn OnMethodEnd<TTarget>(TTarget instance, Exceptio
     private static void EmptyCleanUpMethod()
     {
     }
+
+#pragma warning disable SA1201
+    internal interface ITestAssemblyInfoWithAssembly : ITestAssemblyInfo
+#pragma warning restore SA1201
+    {
+        Assembly Assembly { get; }
```

Comment:
Should this be nullable, or is always non-null?

Reply:
is not nullable in the original code. (it has nullable enabled)